File: class24/cpp/AP2408-ValidaSenha.cpp
#include <iostream>
#include <string>
bool senhaForte(std::string s){return s.size()>=8;}
int main(){std::cout<<"12345678 forte? "<<senhaForte("12345678")<<"\n";std::cout<<"123 forte? "<<senhaForte("123")<<"\n";return 0;}

File: class24/java/AP2408-ValidaSenha.java
public class ValidaSenha {
    static boolean senhaForte(String s) { return s.length() >= 8; }
    public static void main(String[] x) {
        System.out.println("12345678 forte? " + senhaForte("12345678"));
        System.out.println("123 forte? " + senhaForte("123"));
    }
}
